core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/ArithmeticFunction.java

```diff
@@ -14,17 +14,25 @@
 import static org.opensearch.sql.data.type.ExprCoreType.SHORT;
 
 import lombok.experimental.UtilityClass;
+import org.opensearch.sql.data.model.ExprNullValue;
```

Comment:
Why were the imports moved around? Was checkstyle not happy with the original order?

Reply:
The order has been fixed